pages/profile.js

```diff
@@ -54,13 +57,25 @@ export default function Profile(props) {
           <h1 className="py-4 text-4xl font-bold text-gray-darker">
             {t.profileAndSecuritySettings}
           </h1>
-          <div className="my-4 gap-4 flex flex-row sm:flex-row xs:flex-col">
-            <SettingsNavButton text={t.ei} active={true}></SettingsNavButton>
-            <SettingsNavButton text={t.cpp}></SettingsNavButton>
-            <SettingsNavButton text={t.oas}></SettingsNavButton>
-          </div>
-          <ProfileInfo fields={[fakeFields, fakeFields2]} />
-          <Link href="/dashboard">
+          <TabList
+            titles={[t.profileAndPreferences, t.securitySettings]}
+            onClick={(index) => setTabSelected(index)}
+            tabSelected={tabSelected}
+          />
+          {tabSelected === 0 ? (
+            <>
+              <div className="my-4 gap-4 flex flex-row sm:flex-row xs:flex-col">
+                <SettingsNavButton text={t.ei} active></SettingsNavButton>
+                <SettingsNavButton text={t.cpp}></SettingsNavButton>
+                <SettingsNavButton text={t.oas}></SettingsNavButton>
+              </div>
+              <ProfileInfo fields={[fakeFields, fakeFields2]} />{' '}
+            </>
+          ) : tabSelected === 1 ? (
+            <p>Security Settings</p>
```

Comment:
Worth making each tab its own component for organisation

Reply:
Taking care of this in my next PR